src/gallium/drivers/etnaviv/hw/isa.xml.h

```diff
@@ -120,6 +121,14 @@ DEALINGS IN THE SOFTWARE.
 #define INST_SWIZ_COMP_Y					0x00000001
 #define INST_SWIZ_COMP_Z					0x00000002
 #define INST_SWIZ_COMP_W					0x00000003
+#define INST_TYPE_F32						0x00000000
```

Comment:
Maybe split up the commit in two patches:
1) "etnaviv: update generated header files"
2) "etnaviv: Implement primitive restart"